GalaxyZooTouchTable/GalaxyZooTouchTable/Views/ClassificationPanel.xaml

```diff
@@ -25,29 +25,9 @@
                 <RowDefinition Height="13*"/>
             </Grid.RowDefinitions>
 
-            <Grid.ColumnDefinitions>
-                <ColumnDefinition Width="5*"/>
-                <ColumnDefinition Width="1*"/>
-            </Grid.ColumnDefinitions>
-
-            <Border
-                Grid.Row="1"
-                Grid.Column="1"
-                Background="Black"
-                CornerRadius="5,0,0,5"
-                BorderThickness="1"
-                Padding="0">
-
-                <StackPanel>
-                    <Rectangle Fill="{Binding DataContext.ThemeColor, RelativeSource={RelativeSource AncestorType=local:UserConsole}}" Margin="1" Stretch="UniformToFill"/>
-                    <Rectangle Fill="{Binding DataContext.ThemeColor, RelativeSource={RelativeSource AncestorType=local:UserConsole}}" Margin="1" Stretch="UniformToFill"/>
-                    <Rectangle Fill="{Binding DataContext.ThemeColor, RelativeSource={RelativeSource AncestorType=local:UserConsole}}" Margin="1" Stretch="UniformToFill"/>
-                    <Rectangle Fill="{Binding DataContext.ThemeColor, RelativeSource={RelativeSource AncestorType=local:UserConsole}}" Margin="1" Stretch="UniformToFill"/>
-                    <Rectangle Fill="{Binding DataContext.ThemeColor, RelativeSource={RelativeSource AncestorType=local:UserConsole}}" Margin="1" Stretch="UniformToFill"/>
-                </StackPanel>
-            </Border>
+            <Views:Leveler Grid.Row="1"/>
 
-            <Button
+            <!--<Button
```

Comment:
Ah yes, this is commented out because it's pending. The bell pullout might change in the future (along with the "ask a friend feature") but for now I'm commenting it out as a placeholder. I can add a comment as well.